transactor/src/main/scala/io/mediachain/transactor/StateMachine.scala

```diff
@@ -12,6 +13,8 @@ import cats.data.Xor
 
 object StateMachine {
   import io.mediachain.transactor.Types._
+
+  val JournalBlockSize: Int = 4096 // blocksize for Journal Blockchain
```

Comment:
Let this be here, as it is out of scope for this branch. 
I'll fix it in a separate branch as it has dependent changes in the tests too.